website/docs/d/scheduler_job_collection.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_scheduler_job_collection"
+sidebar_current: "docs-azurerm-datasource-scheduler_job_collection"
+description: |-
+  Get information about the specified schelduer job collection.
+---
+
+# Data Source: azurerm_scheduler_job_collection
+
+Use this data source to access the properties of an Azure scheduler job collection.
+
+## Example Usage
+
+```hcl
+data "azurerm_scheduler_job_collection" "test" {
+  name                = "tfex-job-collection"
+  resource_group_name = "tfex-job-collection-rg"
+}
+
+output "job_collection_state" {
+  value = "${data.azurerm_scheduler_job_collection.jobs.state}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Scheduler Job Collection. Changing this forces a new resource to be created.
```

Comment:
We're not creating anything - so we can we change this to `Specifies the name of the Scheduler Job Collection`